### ISSUE TITLE
SIGSEGV on use tensorflow

### ISSUE THREAD

**Issue by mbajlo** [tensorflow/rust#373]
Hello, 
I am have been trying to setup the example [mobilenetv3](https://github.com/tensorflow/rust/blob/master/examples/mobilenetv3.rs). In my `Cargo.toml` I have dependecy `tensorflow = { version = "0.19.0" }`, up to this point I can make `cargo build` and it works fine.
The problem happens if I add any kind of `use tensorflow::`, after this if I do build, even though I haven't used any object from `tensorflow` I get this: `Stop reason: signal SIGSEGV: invalid address (fault address: 0x0)`.

So basically I get `segmentation fault`, which means I want to use some undefined memory even before I used any code from `tensorflow crate`. 
Do you guys know what could be wrong here?

**Comment by adamcrume** [comment]
I'm unable to reproduce the issue, even running under valgrind.  Can you verify which TensorFlow library is getting pulled in?  You can try building with `-v` if it's not clear.  You might also want to try running the example with `gdb` and printing a backtrace (`bt`) when it crashes.

**Comment by mbajlo** [comment]
Thanks for a response, I will try what you have explained.

Now when I look at the code I see I have forgotten to remove async main function under the `tokio` framework. Maybe the problem lies there since there is no `async` support in `tensorflow`?
I just copied the whole example into the
`async main`

**Comment by mbajlo** [comment]
Hi

I tried to make a new binary package in which I will have just the code from the mentioned [example](https://github.com/tensorflow/rust/blob/master/examples/mobilenetv3.rs).
I made the package with `cargo new test_tf --bin` and in Cargo.toml I added `tensorflow` dependency `tensorflow = { version = "0.19.0", features = ["eager"] }`. 
After this I made `cargo build` and tried `cargo run` but I keep getting this:

> error while loading shared libraries: libtensorflow_framework.so.2: cannot open shared object file: No such file or directory

And I have `libtensorflow_framework.so.2` under:

> test_tf/target/debug/build/tensorflow-sys-9b61d66e8ef366b9/out/libtensorflow-cpu-linux-x86_64-2.9.1/lib/libtensorflow_framework.so.2
> test_tf/target/debug/build/tensorflow-sys-9b61d66e8ef366b9/out/libtensorflow_framework.so.2

I do not know why is this happening, from what I have understood, this is a `linker` error, but the linking should be taken care of by `tensorflow` crate?
